ethereum-governance/src/common/constants.ts

```diff
@@ -74,6 +74,8 @@ export const INCREASE_STAKING_LIMIT_ADDRESS =
   "0xfebd8fac16de88206d4b18764e826af38546afe0";
 export const REWARD_PROGRAMS_REGISTRY_ADDRESS =
   "0xfcad241d9d2a2766979a2de208e8210edf7b7d4f";
+export const STONKS_TOP_UP_ALLOWED_RECIPIENTS_CONTRACT =
```

Comment:
Let's update the name to 
```suggestion
export const STONKS_TOP_UP_ALLOWED_RECIPIENTS_ADDRESS =
```
so it will follow the patern.